highlevel/robot/entity/type.py

```diff
@@ -33,3 +36,21 @@ class Direction(Enum):
     BACKWARD = 'BACKWARD'
     LEFT = 'LEFT'
     RIGHT = 'RIGHT'
+
+
+def tick_to_mm(tick: Tick, ticks_per_revolution: Tick,
```

Comment:
Maybe we should move this "type" file out of entities to util...

Reply:
Agreed, I was thinking about doing it.